### ISSUE TITLE
A11y_Windows Template Studio_Setting Details_AI4D: Control type of “Back” button is not correct. (Given Hyperlink)

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/WindowsTemplateStudio#3896]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#GH_WindowsTemplateStudio_Oct2020; #Windows Template Studio;#Win10;#FTP;#DesktopApp;#WCAG4.1.2#AI4D;NonBenchmark;

### **Environment Details:**
Application Name: Windows Template Studio (WinTS)
Templates version: 3.8.20273.1
Wizard version: 3.8.20273.1Date: 2020-06-17T21:13:20.174Z
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.8.0 Preview 3.1
Windows Version: Windows10

### **Repro Steps:**

1. Open Visual Studio 2019 
2. Click on File from menu and then select new and then project.
3. Select the "Windows template studio (WPF.Net Core)”. Click on next
4. "Configure your new project" screen will appear. Click on create button.
5. New WPF App screen will open. Navigate to "Pages" in the left navigation menu with the help of keyboard Tab key and click enter on it.
6. "Add Pages "screen will appear.  Navigate to “Settings” list item and hit enter on details option.
7. Start Ai4D tool and check Control type of “Back” button.

### **Actual:**
Control type of “Back” button is not correct. (Given Hyperlink)

**Same issue is also observed with throughout the application where "Back" button present in detail section.**

### **Expected:**
Control type of “Back” button should be Implemented as “Button” not Hyperlink.

### **User Impact:**
Users will not get correct information if control type is not correct. Screen reader announce Hyperlink due to which user will get confuse.

**Comment by sibille** [comment]
Verified in dev-nightly version:
Wizard version: 0.22.20310.1

You can install the dev-nightly version from our nightly feed (https://www.myget.org/F/windows-template-studio-nightly/vsix/).
For instructions on how install WTS from the dev-nightly feed in Visual Studio see https://github.com/microsoft/WindowsTemplateStudio/blob/dev/docs/getting-started-extension.md#nightly--pre-release-instructions

**Comment by harshgangrade111** [comment]
Issue does not repro in the latest dev-nightly version. Hence closing This bug.
